AutoMad/madDictionary.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Nov  6 02:36:40 2024
+
+@author: dragon
+"""
+
+from dataclasses import dataclass
+import numpy as np
+import pandas as pd
+import random
+import wordFormer
+
+@dataclass
+class madDictionary:
+    dictionary : pd.DataFrame
+    
+    def getWord(self, wordType):
+        for category in self.dictionary.columns:
+            if wordType == category:
+            #print(random.choice(df[category].dropna()))
+                return random.choice(list(self.dictionary[category].dropna()))
+        return wordType
+        
+    def inCategory(self, word, category):
+        if self.dictionary[category].str.fullmatch(word.title()).any():
+            return True
+        return False
+    
+    def getWordInflect(self, wordType, possum):
+        for category in self.dictionary.columns:
+            if wordType == category:
+            #print(random.choice(df[category].dropna()))
+                newWord = random.choice(list(self.dictionary[category].dropna()))
+                newWord = wordFormer.inflectCorrect(newWord, possum)
+                return newWord
+        return wordType
```

Comment:
These variable names leave me a bit confused about the types of what is going on here. This function looks like it could return either a `Word` or a `WordType` which, to my haskell brain at least, look like they might be things of a different "kind". Type annotations and a docstring might help